src/main/java/mate/academy/springintro/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.springintro.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springintro.dto.user.UserRegistrationRequestDto;
+import mate.academy.springintro.dto.user.UserResponseDto;
+import mate.academy.springintro.exception.RegistrationException;
+import mate.academy.springintro.mapper.UserMapper;
+import mate.academy.springintro.model.User;
+import mate.academy.springintro.repository.UserRepository;
+import mate.academy.springintro.service.UserService;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class UserServiceImpl implements UserService {
+    private final UserRepository userRepository;
+    private final UserMapper userMapper;
+
+    @Override
+    public UserResponseDto register(UserRegistrationRequestDto requestDto)
+            throws RegistrationException {
```

Comment:
I can't find handling RegistrationException. Let’s add this logic)

Reply:
fix comment